### ISSUE TITLE
Incorrect COUNT SQL syntax for statement with DISTINCT keyword

### ISSUE THREAD

**Issue by yonisade** [dbeaver/dbeaver#14369]
#### System information: 
- Operating system (distribution) and version: Windows 11 64 bit
- DBeaver version: Community 21.2.3.202110151816
- Java version: 1.8.0_311

#### Connection specification:
- Database name and version: MySQL 5.7.35 / MySQL 8.0.26
- Driver name: MySQL connector 5.1.44 / MySQL connector 8.0.19
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
Getting error "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ALL [date_column]) FROM"
when trying to calculate total row count (green arrows icon) of query:
select distinct [date_column] from [table]

#### Steps to reproduce, if exist:

```sql
create table x (creation_time datetime);

insert into x 
select sysdate()+interval row_number() over () second from information_schema.`COLUMNS`;

select distinct creation_time from x;
```

**Comment by uslss** [comment]
thanks for the report

**Comment by LonwoLonwo** [comment]
Fixed in Dbeaver 21.2.4

**Comment by uslss** [comment]
verified
